principles.qmd

```diff
@@ -23,6 +23,8 @@ The mindmap below provides an overview of the blueprints content. Please refer t
 
 This page is focused on general principles of the project and for each one of these principles, we link to other sections of the book detailing the practical application of these principles in our processes.
 
+Additionally, these principles are implemented in a GitHub template repository: [epiverse-trace/packagetemplate](https://github.com/epiverse-trace/packagetemplate). This template repository is aiming to facilitate the adoption of the standards presented here and provide a quick start point for new packages. Further information about this template repository is available in the [dedicated wiki](https://github.com/epiverse-trace/packagetemplate/wiki).
```

Comment:
I think liking the GH docs on template repositories would be good for people that are not aware of this feature (https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).

Reply:
> Would under the "GitHub template repository" first mention be a good place to link this?

Yes that works.